### PR TITLE
Update Stable Manifest(main branch)

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
#### PR checklist

Checklist items below are applicable for development targeted to both fast and stable branches/tags
- [ ] Unit tests pass locally
- [ ] FVT tests pass locally
- [ ] If the PR adds a new container image or updates the tag of an existing image (not build within cpaas), is the corresponding change made in live-builder and cpaas-midstream to add/update the image tag in the operator CSV? Link the PRs if applicable

Checklist items below are applicable for development targeted to both fast and stable branches/tags
- [ ] Tested modelmesh serving deployment with odh-manifests and ran odh-manifests-e2e tests locally

### CODE REVIEW

File: opendatahub/odh-manifests/model-mesh_stable/odh-model-controller/overlays/odh/odh_model_controller_manager_patch.yaml [hunk 1]
```diff
@@ -11,11 +11,6 @@ spec:
             - --leader-elect
             - "--monitoring-namespace"
             - "$(MONITORING_NS)"
-#This is hardcoded to be false in the overlay to model-mesh. 
-# Once Kserve and Modelmesh CRD conflict is resolved and both components can be installed together, 
-# ODH Model Controller will not be deployed as a overlay anymore and will be deployed only as an independent component. 
-# When that happens, the "kserve-enabled" flag needs to be removed
-            - --kserve-enabled=false
```
Comment (VedantMahabaleshwarkar) [inline opendatahub/odh-manifests/model-mesh_stable/odh-model-controller/overlays/odh/odh_model_controller_manager_patch.yaml:18]: Why is this removed? This is needed for odh-model-controller when used with modelmesh.

**Review by VedantMahabaleshwarkar — CHANGES_REQUESTED** [review]

File: opendatahub/odh-manifests/model-mesh_stable/odh-model-controller/overlays/odh/odh_model_controller_manager_patch.yaml (hunk 1 above)
Comment (vaibhavjainwiz) [reply opendatahub/odh-manifests/model-mesh_stable/odh-model-controller/overlays/odh/odh_model_controller_manager_patch.yaml:18]: Now modelmesh-serving and kserve could be deployed togeather. I had tested it on my system and its working.

Comment (vaibhavjainwiz) [reply opendatahub/odh-manifests/model-mesh_stable/odh-model-controller/overlays/odh/odh_model_controller_manager_patch.yaml:18]: added back

**Review by VedantMahabaleshwarkar — APPROVED** [review]
/lgtm

**Comment by Jooho** [comment]
This pr does not need to pass openshift ci `fvt-odh-manifests` because this pr is for stable manifests. 
OpenShift ci for main branch is using fast manifests.
